src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,162 @@
+package duke;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import duke.exception.DukeException;
+import duke.helper.Parser;
+import duke.helper.Storage;
+import duke.helper.TaskList;
+import duke.helper.Ui;
+import java.io.IOException;
```

Comment:
Would it be better to split up your `import` statements? 

You can refer to [this document](https://se-education.org/guides/tutorials/intellijCodeStyle.html#tweak-import-order) to tweak your import order in IntelliJ IDEA.